src/utils.ts

```diff
@@ -39,3 +42,25 @@ export function allFalseOnKey(obj: any, key: any) {
     }
     return true;
 }
+
+/**
+ * Get X axis position value for supplied object
+ * @param {sting} position - Position prop, Accepted values right, left, center
+ * @param {number} childWidth - Width of child element in pixels, for center and left alignment
+ * @returns {number} x axis position
+ * */
+
+export const getXPositionValue = (position: string, childWidth: number): number => {
```

Comment:
If we are using this function only for datagrid filter then please move it into that file